react/components/EXPERIMENTAL_Table/util.ts

```diff
@@ -0,0 +1,29 @@
+export const constants = {
```

Comment:
Export to external constants file. I'm guessing to will keep on growing.

Reply:
Done